ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -651,6 +653,42 @@ export default class AmendRegistration extends Vue {
     this.debtorValid = val
   }
 
+  private validateSecuredParties (): void {
+    const sp = this.getAddSecuredPartiesAndDebtors.securedParties
+    let securedPartyCount = 0
+    if (this.registrationType === APIRegistrationTypes.SECURITY_AGREEMENT) {
+      for (let i = 0; i < sp.length; i++) {
+        // is valid if there is at least one secured party
+        if (sp[i].action !== ActionTypes.REMOVED) {
+          securedPartyCount++
+        }
+      }
+      if (securedPartyCount >= 1) {
+        this.setValidSecuredParties(true)
+      } else {
+        this.setValidSecuredParties(false)
+      }
```

Comment:
This could be simplified to `this.setValidSecuredParties(securedPartyCount >= 1)`

Reply:
Nice.  Made change...